eng/pipelines/libraries/helix.yml

```diff
@@ -3,6 +3,7 @@ parameters:
   archType: ''
   buildConfig: ''
   creator: ''
+  condition: ''
```

Comment:
condition is already defined in L12 of this template. Please remove this addition.